cli/util/upload_bin.go

```diff
@@ -208,13 +209,12 @@ func fetchLatestMissingBlockIndex(ctx context.Context, p *pool.Pool, containerID
 			}
 
 			wg.Add(1)
-			go func(i, startIndex, endIndex int) {
+			go func(i, startIndex int) {
```

Comment:
Revert it, replace with `if endIndex == startIndex + 1 { // search with object.MatchStringEqual } else { // search with GE/LE}` and appropriate comment.